internal/cluster/cluster.go

```diff
@@ -6,9 +6,10 @@ import (
 	"strings"
 
 	// Reusing the Prometheus Ha Exporter cibadmin xml parser here
```

Comment:
This comment looks a leftover. If we want to have a referent I would point it out with the url of the project just in top of the 2 imports